router/router.go

```diff
@@ -855,27 +850,22 @@ func (worker *workerT) processDestinationJobs() {
 			userToJobIDMap[destinationJobMetadata.UserID] = destinationJobMetadata.JobID
 		}
 
-		if attemptedToSendTheJob {
-			status.AttemptNum++
-		}
-
+		status.AttemptNum++
```

Comment:
The only risk I see by removing this check is, if transformer throws a 5xx for a valid job (because of some bug), we would end up aborting after certain number of retries.

This was introduced to help in those scenarios. I am not sure if there is a proper alert mechanism to identify these bugs.

Reply:
Any thoughts. @cisse21 @atzoum

Reply:
Even now we don't know this happens, so we could add an alert for > HTTP 500 errors?

Reply:
Now, we will know if there is a growth in router jobs queue. If in any case, the queue doesn't grow, you are right. We aren't alerted.

Integrations team might already have alerts on router_response metrics. I am not completely sure though.

I am okay with this change. Just want to highlight above.